org.gnome.Games.json

```diff
@@ -0,0 +1,147 @@
+{
+    "app-id": "org.gnome.Games",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "branch": "stable",
+    "sdk": "org.gnome.Sdk",
+    "command": "gnome-games",
+    "tags": ["nightly"],
+    "finish-args": [
+        /* Extension point */
+        "--extension=org.gnome.Games.LibretroPlugin=directory=lib/libretro/extra.d",
+        "--extension=org.gnome.Games.LibretroPlugin=subdirectories",
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* Wayland access */
+        "--socket=wayland",
+        /* PulseAudio access */
+        "--device=dri", "--socket=pulseaudio",
+        /* Tracker access */
+        "--talk-name=org.freedesktop.Tracker1",
+        /* Needed for dconf to work */
+        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        /* Needed to get the games info from The Games DB */
+        "--share=network",
+        /* Needed for joypads, until we have a proper portal */
+        "--device=all",
+        /* Needed to find the ROMs */
+        "--filesystem=home:ro"
+    ],
+    "build-options" : {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g"
+    },
+    "cleanup": ["/include", "/lib/pkgconfig",
+                "/share/pkgconfig", "/share/aclocal",
+                "/man", "/share/man", "/share/gtk-doc",
+                "*.la", "*.a",
+                "/lib/girepository-1.0",
+                "/share/dbus-1", "/share/doc", "/share/gir-1.0"
+    ],
+    "modules": [
+        /* Emulator cores for libretro
+
+           We build them first, because some of them are very big, and they
+           change more rarely than the rest of the modules. */
+        "libretro-cores/libretro-bsnes.json",
+        "libretro-cores/libretro-gambatte.json",
+        "libretro-cores/libretro-mednafen_ngp.json",
+        "libretro-cores/libretro-mednafen_pce_fast.json",
+        "libretro-cores/libretro-nestopia.json",
+        "libretro-cores/libretro-pcsx_rearmed.json",
+        /* End libretro cores */
+        "org.lua.modules.json",
```

Comment:
Can use https://github.com/flathub/shared-modules/tree/master/lua5.3 to reduce duplication.